WriteVHDLSyntax.py

```diff
@@ -719,10 +858,12 @@ def writeTBConstants(memDict, memInfoDict, procs, emData_dir, sector):
     string_constants += "  constant inputFileNameEnding".ljust(str_len) + ": string := \"_" + sector + ".dat\"; -- " + sector + " specifies the nonant/sector the testvectors represent\n"
     string_constants += "  constant outputFileNameEnding".ljust(str_len) + ": string := \".txt\";\n"
     string_constants += "  constant debugFileNameEnding".ljust(str_len) + ": string := \".debug.txt\";\n\n"
+    string_constants += "  signal dummy : STD_LOGIC := '0';\n\n -- dummy tb signal for inputs into sectorproc\n"
+    string_constants += "  signal dummyaddr : t_as_36_addr := (others => '0');\n\n -- dummy tb signal for inputs into sectorproc"
```

Comment:
@jasonfan393 Maybe it's worth defining a dedicated type for `dummyaddr` here, to avoid using `t_as_36_addr`, which seems random.

Reply:
this dummyaddr is really only to be used for the AS_36 read addresses in the split project. These dummy signals should ultimately be removed when we properly add the VMSMERouter to the wiring config files, but for now it is just a quick fix to have a valid testbench.  Maybe I can change the name of this to something like "dummyASReadAddr" to make its purpose more clear, and add a fixme comment?

Reply:
Thanks for explaining. Yeah, maybe just add a FIXME comment.